assets/js/app.js

```diff
@@ -57,7 +57,20 @@ Hooks.CommandInput = {
 
 let csrfToken = document.querySelector('meta[name=\'csrf-token\']').getAttribute('content');
 /* eslint-disable-next-line camelcase */
-let liveSocket = new LiveSocket('/live', Socket, {hooks: Hooks, params: {_csrf_token: csrfToken}});
+let liveSocket = new LiveSocket(
+  '/live',
+  Socket,
+  {
+    hooks: Hooks,
+    params: {_csrf_token: csrfToken},
+    metadata: {
+      keydown: (e, el) => {
+        return { code: e.code };
```

Comment:
It looks like we're already getting this value under the key "key", it's probably a default for this event so there's no need to send it manually

Reply:
Great call - thanks!